api/v1alpha1/clusterextension_types.go

```diff
@@ -23,8 +23,7 @@ import (
 )
 
 var (
-	ClusterExtensionGVK  = SchemeBuilder.GroupVersion.WithKind("ClusterExtension")
```

Comment:
`ClusterExtensionGVK` is unused, and is not really necessary for inclusion as an exported global variable in the API.

Callers can construct themselves like this:
```go
gvk := ocv1alpha1.GroupVersion.WithKind(v1alpha1.ClusterExtensionKind)
```